interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is a local operating system that logs past file changes. Github is an online version of git for programmers to work together.
 
-  Researched answer:
+  Researched answer: Git is a control system to manage different programming projects. Git is a local source and Github is an online source
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: Greater than, equal to, less than will return a Boolean value.
 
-  Researched answer:
+  Researched answer:Greater than, equal to, less than will return a Boolean value. More specifically loose equality (==), and strict equality (===)
```

Comment:
Awesome!  There are also logical operators such as && and || that will return a Boolean value.